app/controllers/setup_controller.rb

```diff
@@ -107,6 +107,9 @@ def settings_params
       settings["no_proxy"] = ""
     end
 
+    settings["dex_client_secrets_kubernetes"] = Pillar.value(pillar: :dex_client_secrets_kubernetes) || SecureRandom.uuid
+    settings["dex_client_secrets_velum"] = Pillar.value(pillar: :dex_client_secrets_velum) || SecureRandom.uuid
```

Comment:
Can these cases really happen? We should have migrated the database, no?

Reply:
The migrations don't run on a fresh install?

Reply:
Right.